src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -20,13 +20,19 @@ const Authenticate = (() => {
                     if (!/authenticated/.test(status)) {
                         init();
                         $('#not_authenticated').setVisibility(1);
-                        let link = 'https://marketing.binary.com/authentication/2017_Authentication_Process.pdf';
-                        if (Client.isAccountOfType('financial')) {
-                            $('#not_authenticated_financial').setVisibility(1);
-                            link = 'https://marketing.binary.com/authentication/2017_MF_Authentication_Process.pdf';
+                        // Show upload instructions
+                        if (jpClient()) {
+                            $('.jp-show').setVisibility(1);
```

Comment:
I think you can use existing classes `ja-hide` and `ja-show`, js code already handles hiding and showing those

Reply:
@negarn Doesn't seem to work.